ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantStateReaderSpec.scala

```diff
@@ -265,5 +265,7 @@ object KeyValueParticipantStateReaderSpec {
       reader,
```

Comment:
Please add a test case documenting what happens when `failOnUnexpectedEvent = false` and we're not receiving a log entry.

Reply:
Done.